plugin/build.gradle

```diff
@@ -238,6 +261,7 @@ configurations.all {
     resolutionStrategy.force 'junit:junit:4.12'
     resolutionStrategy.force 'org.apache.commons:commons-lang3:3.10'
     resolutionStrategy.force 'commons-logging:commons-logging:1.2'
+    resolutionStrategy.force 'org.objenesis:objenesis:3.2'
```

Comment:
Hey @Zhangxunmt this change is not part of plugin [document](https://github.com/opensearch-project/opensearch-plugins/blob/main/BUILDING.md#plugin-usage), can you please check?

Reply:
Hi @prudhvigodithi, thanks for your comment! Yes it's not for this "publish to maven" change, but we will need it anyways in our next feature for adding a new algorithm dependency. It's no harm to include it here before the next PR of adding that algorithm.

Reply:
Got it, thanks @Zhangxunmt